src/engine/openspaceengine.cpp

```diff
@@ -1104,12 +1110,25 @@ void OpenSpaceEngine::preSynchronization() {
     if (_hasScheduledAssetLoading) {
         LINFO(fmt::format("Loading asset: {}", absPath(_scheduledAssetPathToLoad)));
         global::profile->ignoreUpdates = true;
-        loadSingleAsset(_scheduledAssetPathToLoad);
+        loadAsset_init(_scheduledAssetPathToLoad);
         global::profile->ignoreUpdates = false;
         resetPropertyChangeFlagsOfSubowners(global::rootPropertyOwner);
         _hasScheduledAssetLoading = false;
         _scheduledAssetPathToLoad.clear();
     }
+    else if (!_hasInitializedProfile) {
```

Comment:
Isn't `_hasInitializedProfile` the same as `_isFirstRenderingFirstFrame` (whose name probably should be `_isRenderingFirstFrame` instead) 🤔

Reply:
They mean different things, but true that `_isRenderingFirstFrame` will accomplish both, so made that change and deleted `_hasInitializedProfile`.

Reply:
I wasn't arguing for that they *should* be the same, just wondering if there *were* :)